### PR TITLE
Fix searching built-in operators

### PR DESCRIPTION
**Pull request by ppisar**
This fixes <https://rt.cpan.org/Ticket/Display.html?id=126015>.

### CODE REVIEW

**Comment by briandfoy** [comment]
I've just taken over the maintenance of Pod::Perldoc, so I'll look into this. I know it's years old, and it might take me a bit to catch up.

**Comment by haarg** [comment]
I think it would be better to include only a representative fragment of perlop for testing this. Including the full document will be similar to perlfunc and cause continual confusion when this is brought into core.

**Comment by briandfoy** [comment]
> I think it would be better to include only a representative fragment of perlop for testing this. Including the full document will be similar to perlfunc and cause continual confusion when this is brought into core.

Can you expound on that? Is it the filename that's the problem, or is something looking at the contents? Are there any links to previous threads?

**Comment by briandfoy** [comment]
Okay, I think I know what is going on here. I thought it was all related and that confused me.

* there's a test for the tr issue from https://rt.cpan.org/Public/Bug/Display.html?id=86506
* unrelated to that, there's a fix that restored `-f` looking in perlop.

Both of these seem to work for me, so I'll merge this fix.

File: Makefile.PL [hunk 1]
```diff
@@ -15,6 +15,7 @@ WriteMakefile(
 
     'PREREQ_PM' => {
             # Are there any hard dependencies not covered here?
+            'blib'                      => '0',
```
Comment (briandfoy) [inline Makefile.PL:18]: No one appears to use this, so I might remove it after I merge.

**Review by briandfoy — APPROVED** [review]
Looks good.

**Comment by briandfoy** [comment]
I've rebased this and re-push as #70.

**Pull request closed without merging**